components/constellation/sandboxing.rs

```diff
@@ -264,6 +247,31 @@ trait CommandMethods {
         U: AsRef<OsStr>;
 }
 
+#[cfg(all(
+    not(target_os = "windows"),
+    not(target_os = "ios"),
+    not(target_os = "android"),
+    not(target_env = "ohos"),
+    not(target_arch = "arm"),
+    not(target_arch = "aarch64")
+))]
+impl CommandMethods for gaol::sandbox::Command {
```

Comment:
Not really fan of duplicating complex cfgs, we should put all this stuff in module to prevent lint from triggering or simply allow lint.

Reply:
Hmm... I think I would prefer to duplicate. But I've put it back and allowed the lint for now. A module will require a duplicated cfg to use the items anyway.

I think the cfg'ing in the module could be simplified in general. So perhaps we can sort that out all at once.